ExampleMod/Content/ExampleRecipes.cs

```diff
@@ -21,6 +21,10 @@ public static class ExampleRecipes
 			Examples_AddRecipes(mod);
 
 			Examples_ModifyRecipes(mod);
+
+			// See RecipeHelper.cs
+			RecipeHelper.AddExampleRecipes(mod);
+			RecipeHelper.ExampleRecipeEditing(mod);
```

Comment:
Would probably make more sense to make use of an `ILoadable`'s `Load()` or `ModSystem`'s `OnModLoad()` to do this.